vite.config.js

```diff
@@ -1,7 +1,11 @@
 import { defineConfig } from 'vite'
 import react from '@vitejs/plugin-react'
 
-// https://vitejs.dev/config/
+const { PORT } = process.env
+
 export default defineConfig({
   plugins: [react()],
+  server: {
+    port: PORT || 5173
```

Comment:
Would you want to have this default to 5175 or something other than the port you just swapped from?